### ISSUE TITLE
Func: render_label  means3D.detach()

### ISSUE THREAD

**Issue by wkMvg** [fzhiheng/RoGS#8]
means3D.detach Does it mean that semantic information is not used to supervise the ground point cloud? beg your reply

**Comment by fzhiheng** [comment]
Yes, in the original design, semantics is not used to supervise the ground. In practice, you could try cancelling `detach()`.

**Comment by christianll9** [comment]
> in the original design, semantics is not used to supervise the ground

Is there a reason for it? Decrease in performance?
